scss/colors.scss

```diff
@@ -0,0 +1,3 @@
+$celerean: #00ADEE;
```

Comment:
For now i would use `Blue` `Green` `Grey` because be only have these few colors. Now if we had many colors like 3 blue colors... 4 red colors it would be a good idea to write them more specifically like you did